Makefile

```diff
@@ -15,6 +15,8 @@ OS := $(strip ${os})
 ARCH := $(shell uname -m)
 ifeq ($(ARCH),x86_64)
 	ARCH := amd64
+else ifeq ($(ARCH),aarch64)
+	ARCH := arm64
 endif
```

Comment:
While we fixing it, let's use `go env GOARCH` instead of `uname -m` + `if`. This starting to get too ugly.

Reply:
I've been thinking only about `ARCH := $(go env GOARCH)`, but if you think about other check to add sure add them

Reply:
GOHOST* seems like a better choice in this specific case, as we want to know host OS data and not cross-compile.

Reply:
Got it. Sure, let's use `GOHOSTARCH` then

Reply:
fixed